setup/ansible-inventory

```diff
@@ -45,40 +45,3 @@ iojs-build-centos5
 
 [iojs-aix61]
 test-ibm-aix61-ppc64-1:18822
-
-[node-windows]
-node-win10-x64-1.cloudapp.net
```

Comment:
Just to confirm, all of these inventory entries exist in `ansible/inventory.yml` (ignoring whether or not the Ansible playbook has been run on them) Don't want these hosts to get lost somehow